lib/android/src/main/java/com/airbnb/android/react/maps/AirMapPolylineManager.java

```diff
@@ -4,6 +4,7 @@
 import android.graphics.Color;
 import android.os.Build;
 import android.util.DisplayMetrics;
+import android.util.Log;
```

Comment:
@kruyvanna please remove this import

once done I'm more than happy to test and merge